addons/web/views/webclient_templates.xml

```diff
@@ -116,13 +116,15 @@
         <link rel="stylesheet" type="text/less" href="/web/static/src/less/list_view_extra.less"/>
         <link rel="stylesheet" type="text/less" href="/web/static/src/less/kanban_dashboard.less"/>
         <link rel="stylesheet" type="text/less" href="/web/static/src/less/kanban_view.less"/>
+        <link rel="stylesheet" type="text/less" href="/web/static/src/less/kanban_view_mobile.less"/>
         <link rel="stylesheet" type="text/less" href="/web/static/src/less/web_calendar.less"/>
         <link rel="stylesheet" type="text/less" href="/web/static/src/less/search_view.less"/>
         <link rel="stylesheet" type="text/less" href="/web/static/src/less/search_view_extra.less"/>
         <link rel="stylesheet" type="text/less" href="/web/static/src/less/modal.less"/>
         <link rel="stylesheet" type="text/less" href="/web/static/src/less/data_export.less"/>
 
         <script type="text/javascript" src="/web/static/lib/jquery.scrollTo/jquery.scrollTo.js"></script>
+        <script type="text/javascript" src="/web/static/lib/jquery.touchSwipe/jquery.touchSwipe.js"></script>
```

Comment:
I disagree. I think that we should lazy load the touchSwipe library, not put it in the asset bundles.  You should override the method _loadLibs from the KanbanView to make it load the library with ajax (look at how it is done in _loadLibs in abstract_view)